airflow/api_connexion/openapi/v1.yaml

```
@@ -368,6 +368,8 @@ paths:
       summary: Test a connection
       description: |
         Test a connection.
+        Testing Connections is disabled by default set test_connection flag to enabled in Airflow configuration (airflow.cfg).
+        It can also be controlled by the environment variable AIRFLOW__CORE__TEST_CONNECTION.
```

```suggestion
      description: >
        Test a connection.

        Testing Connections is disabled by default set test_connection flag to enabled in Airflow configuration (airflow.cfg).
        It can also be controlled by the environment variable `AIRFLOW__CORE__TEST_CONNECTION`.
```

This fixes the rendering issues in the generated ts comment. (Don’t apply this on GitHub, you need to do this locally to go through pre-commit)